janusgraph-test/src/test/java/org/janusgraph/graphdb/GraphDatabaseConfigurationInstanceIdTest.java

```diff
@@ -51,35 +45,34 @@ public void graphShouldOpenWithSameInstanceId() {
         final StandardJanusGraph graph1 = new StandardJanusGraph(new GraphDatabaseConfigurationBuilder().build(new CommonsConfiguration(config)));
 
         assertEquals(graph1.openManagement().getOpenInstances().size(), 1);
-        assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique");
+        assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique(current)");
 
         final StandardJanusGraph graph2 = new StandardJanusGraph(new GraphDatabaseConfigurationBuilder().build(new CommonsConfiguration(config)));
 
         assertEquals(graph1.openManagement().getOpenInstances().size(), 1);
-        assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique");
+        assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique(current)");
         assertEquals(graph2.openManagement().getOpenInstances().size(), 1);
-        assertEquals(graph2.openManagement().getOpenInstances().toArray()[0], "not-unique");
+        assertEquals(graph2.openManagement().getOpenInstances().toArray()[0], "not-unique(current)");
         graph1.close();
         graph2.close();
     }
 
     @Test
+    @Disabled("This should be run against a backend which support multiple backends.")
     public void graphShouldNotOpenWithSameInstanceId() {
         final Map<String, Object> map = new HashMap<String, Object>();
         map.put(STORAGE_BACKEND.toStringWithoutRoot(), "inmemory");
         map.put(UNIQUE_INSTANCE_ID.toStringWithoutRoot(), "not-unique");
         final MapConfiguration config = new MapConfiguration(map);
-        final StandardJanusGraph graph1 = new StandardJanusGraph(new GraphDatabaseConfigurationBuilder().build(new CommonsConfiguration(config)));
-
-        assertEquals(graph1.openManagement().getOpenInstances().size(), 1);
-        assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique");
-
-        thrown.expect(JanusGraphException.class);
```

Comment:
Notice that the exception is expected after `graph1` is successfully opened and asserts executed successfully. 
In you code in case when the exception is on opening `graph1` then the unit test won't fail (even so it should fail in such scenario) and asserts won't be executed. Please rewrite you code like this, so that it would be consistent to the old flow:
```
final StandardJanusGraph graph1 = new StandardJanusGraph(new GraphDatabaseConfigurationBuilder().build(new CommonsConfiguration(config)));
assertEquals(graph1.openManagement().getOpenInstances().size(), 1);
assertEquals(graph1.openManagement().getOpenInstances().toArray()[0], "not-unique");
assertThrows(JanusGraphException.class, () -> {
    final StandardJanusGraph graph2 = new StandardJanusGraph(new GraphDatabaseConfigurationBuilder().build(new CommonsConfiguration(config)));
    graph1.close();
}, "A JanusGraph graph with the same instance id [not-unique] is already open. Might required forced shutdown.");
```